conseil-common/src/main/scala/tech/cryptonomic/conseil/common/cache/MetadataCaching.scala

```diff
@@ -15,12 +15,23 @@ object MetadataCaching {
   final case class CacheEntry[A](lastUpdated: LastUpdated, value: A)
 
   /** Class representing key in cache */
-  final case class CacheKey(key: String)
-
-  type Cache[A] = Map[CacheKey, CacheEntry[A]]
-  type AttributesCache = Cache[List[Attribute]]
-  type AttributeValuesCache = Cache[RadixTree[String, String]]
-  type EntitiesCache = Cache[List[Entity]]
+  sealed trait CacheKey {
+    val value: String
```

Comment:
I find this confusing... a key which contains a value...

Is it possible to think of something else if `key` won't work? Like `get`, `text`... don't know

Reply:
`get` works better, but previously there was `key` inside `CacheKey` which was even more confusing to me :D You could access this like `key.key`.